### PR TITLE
Allow props override in agendaList

### PR DESCRIPTION
**Pull request by belgamo**
## Motivation
I wasn't able to customize the prop `renderSectionHeader`. 

## Solution
Put spread of props as the last prop of `<SectionList />`

### CODE REVIEW

**Comment by belgamo** [comment]
> Maybe a spec?
Maybe an units specs would be great to check if props overriding is working, but this project just do e2e tests.

File: src/expandableCalendar/agendaList.js [hunk 1]
```diff
@@ -158,7 +177,10 @@ class AgendaList extends Component {
         onScroll={this.onScroll}
         onMomentumScrollBegin={this.onMomentumScrollBegin}
         onMomentumScrollEnd={this.onMomentumScrollEnd}
-        onScrollToIndexFailed={(info) => { console.warn('onScrollToIndexFailed info: ', info); }}
+        onScrollToIndexFailed={(info) => {
+          console.warn('onScrollToIndexFailed info: ', info);
+        }}
+        {...this.props}
```
Comment (Inbal-Tish) [inline src/expandableCalendar/agendaList.js:183]: This is risky and should be located here. Please specify the need to override in the description.

File: src/expandableCalendar/agendaList.js [hunk 2]
```diff
@@ -25,13 +24,17 @@ class AgendaList extends Component {
     /** day format in section title. Formatting values: http://arshaw.com/xdate/#Formatting */
     dayFormat: PropTypes.string,
     /** style passed to the section view */
-    sectionStyle: PropTypes.oneOfType([PropTypes.object, PropTypes.number, PropTypes.array])
-  }
+    sectionStyle: PropTypes.oneOfType([
```
Comment (Inbal-Tish) [inline src/expandableCalendar/agendaList.js:27]: Please refrain from formatting the file using your own tools and preferences, we are trying to keep the code unified.

File: src/expandableCalendar/agendaList.js (hunk 1 above)
Comment (Inbal-Tish) [reply src/expandableCalendar/agendaList.js:183]: If you need 'renderSectionHeader' why no exposing a prop?

**Pull request closed without merging**